README.md

```diff
@@ -189,11 +208,11 @@ line endings, as it will cause lint errors. For git, please set
 Certificates
 ============
 
-By default pregenerated certificates for the web-platform.test domain
+By default generated certificates for the web-platform.test domain
```

Comment:
Again existing text is right. (Potentially should be hyphenated, but the "correction" is definitely wrong.)